perfer.config.js

```diff
@@ -42,7 +42,7 @@ const files = [
   ['Pagination.bundle.min.js', 10.1],
   ['Picker.bundle.min.js', 9],
   ['ProgressBar.bundle.min.js', 6],
-  ['RadioButton.bundle.min.js', 7.1],
+  ['RadioButton.bundle.min.js', 7.2],
```

Comment:
Maybe we should take a buffer of 1kb.

Reply:
That's a great question, I guess we still don't know how to measure how each kb effects our TPA's